packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+- Choosing an authenticator, such as Okta Verify or WebAuthN.
+- Entering a One Time Passcode (OTP).
+- Cancelling the sign-in.
+
+## Sign-in objects
+
+Each step in the flow is represented by a _Response_ which contains the information used
+for creating the UI and general state information. The response is also used to cancel the
+sign-in, or retrieve an access token after the sign-in succeeds.
+
+The UI information is divided between two objects in the response, authenticators and
+remediations. An _Authenticator_ represents a type, or factor for verifying the identity
+of a user, such as a username and password, or Okta Verify. A _Method_ represents a
+channel for an authenticator, such as using email or SMS for an OTP. An authenticator may
+have multiple methods. A _Remediation_ represents a user action and usually contains most
+of the information for a step. Each response may contain multiple remediations and
+authenticators.
+
+Some common types of user action are represented by a _Capability_. These include
+requesting a new OTP, or a password reset request. Remediations, authenticators, and
+methods may contain capabilities.
+
+A _Form_ inside the remediation represents the user action in a collection of _Field_
+objects. Most fields represent something that's displayed in the UI, either a static
+element, such as a label, or a user input, such as a selection list. The field also
+contains state information, such as whether the associated value is required or secret.
+Options, or lists of choices, are represented by a collection of fields. A field may also
+contain a form which contains more fields.
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+## Common parts of the flow
+
+Some parts of the sign-in flow are the same:
+
+- Add the SDK to your app.
+- Configure the SDK.
+- Initialize the client and start the flow.
+- Process the response.
+- Request a token.
+- Sign-out the user.
+
+### Add the SDK
+
+<StackSnippet snippet="adddependency" />
+
+### Create and manage configurations
+
+A _Configuration_ contains the settings used by the SDK to connect to an Okta Application Integration:
+
+| Value         | Description |
+| :------------ | :---------- |
+| Issuer        | The Oauth2 URL for the Okta Org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin console.  |
+| Client secret | An optional shared secret for accessing the Application Integration. |
```

Comment:
Client secrets should not be used in mobile!!

Reply:
I don't mind one way or the other. I included it because it's explicitly mentioned in the SDK. @mikenachbaur-okta, comments?

I'll go with whatever you folks choose.

Reply:
It's available in the swift SDK for usage on the backend via linux or similar. If we're gearing this towards mobile, it should be strictly prohibited.

Reply:
I've removed the client secret from both the table and the iOS snippet.